x-pack/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/three_way_diff/final_edit/fields/eql_query/eql_query_edit_form.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import type { Filter } from '@kbn/es-query';
+import type { FormData, FormSchema } from '../../../../../../../../shared_imports';
+import { RuleFieldEditFormWrapper } from '../rule_field_edit_form_wrapper';
+import type { FieldValueQueryBar } from '../../../../../../../rule_creation_ui/components/query_bar';
+import {
+  type DiffableRule,
+  RuleEqlQuery,
+  QueryLanguageEnum,
+} from '../../../../../../../../../common/api/detection_engine';
+import { EqlQueryEditAdapter } from './eql_query_edit_adapter';
+// import {
+//   debounceAsync,
+//   eqlValidator,
+// } from '@kbn/security-solution-plugin/public/detection_engine/rule_creation_ui/components/eql_query_bar/validators';
+
+export function EqlQueryEditForm(): JSX.Element {
+  return (
+    <RuleFieldEditFormWrapper
+      component={EqlQueryEditAdapter}
+      ruleFieldFormSchema={kqlQuerySchema}
+      deserializer={deserializer}
+      serializer={serializer}
+    />
+  );
+}
+
+const kqlQuerySchema = {
```

Comment:
Since there are no validations we can probably remove the schema

Reply:
`RuleFieldEditFormWrapper`'s `ruleFieldFormSchema` property is required. We still should pass an empty object. So removing it completely it's possible.